### PR TITLE
[TFWav2Vec2Model] Fix input shapes in TFWav2Vec2WeightNormConv1D

### PR DESCRIPTION
**Pull request by anton-l**
# What does this PR do?
**Context**
After updating to TF 2.7 the tests started failing with `ValueError('One of the dimensions in the output is <= 0...')` which is caused by the recently added check to `keras.layers.Conv.build()` https://github.com/keras-team/keras/blob/v2.7.0/keras/layers/convolutional.py#L197

This PR fixes the input shapes passed during `TFWav2Vec2WeightNormConv1D.build()` to account for padding.

### CODE REVIEW

**Review by patrickvonplaten — APPROVED** [review]

**Review by Rocketknight1 — APPROVED** [review]
LGTM!

**Review by sgugger — APPROVED** [review]
Thanks for the fix!
